From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@comnam90__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@comnam90 : Thanks for your contribution to the Azure documentation! The author, @aungoo-msft, has been notified to review your proposed change.

aungoo-msft commented:
Hi comnam90, Thanks for your feedback. The statement is correct but confusing. We will update this from:
Your application has done 495 I/O units of 16 KB size in one second on a P10 disk. These will be counted as 495 I/O Units per second (IOPS). If you try a 2 MB I/O in the same second, the total of I/O units is equal to 495 + 8. This is because 2 MB I/O results in 2048 KB / 256 KB = 8 I/O Units when the I/O unit size is 256 KB. Since the sum of 495 + 8 exceeds the 500 IOPS limit for the disk, throttling occurs.

To:

Your application has done 495 I/O units of 16 KB size in one second on a P10 disk. These will be counted as 495 I/O Units per second (IOPS). During the same second, if your application sends an additional 2 MB I/O in the same second, the total of I/O units is equal to 495 + 8. This is because 2 MB I/O results in 2048 KB / 256 KB = 8 I/O Units when the I/O unit size is 256 KB. Since the sum of 495 + 8 exceeds the 500 IOPS limit for the disk, throttling occurs.

I hope that will make it clear.

tamram commented:
#please-close

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix2

v-viinde committed:
USER STORY 1237081 SaaS App Tutorial: Shmoop For Schools Metadata URL Update

v-viinde committed:
USER STORY 1237076 SaaS App Tutorial: QPrism Metadata URL Update

anil0225 committed:
Updated Note to Postion Map

anil0225 committed:
Remove Spacings

anil0225 committed:
Initial Commit for documentation

anil0225 committed:
update limitations

anil0225 committed:
Updating Redirections and TOC

anil0225 committed:
Update Tables and Redirections

anil0225 committed:
Reverting Redirection

anil0225 committed:
Remove Redirections

anil0225 committed:
Updated Limitations

lnalepa committed:
Updated v2-app-registration doc

anil0225 committed:
Renamed files

anil0225 committed:
Update refer to

anil0225 committed:
Updated wording

anil0225 committed:
Updated based on recommendations

lnalepa committed:
Merge branch 'master' into lesia-v2-app-reg

anil0225 committed:
Updated Images and Images

anil0225 committed:
Updated screenshot and correction

anil0225 committed:
Updated Mappings

anil0225 committed:
Merged

anil0225 committed:
Updated limitations

anil0225 committed:
Connector Limitations Section

anil0225 committed:
Reverting Image

anil0225 committed:
Updating Image for mappings

anil0225 committed:
Update Tenant

anil0225 committed:
Updated Limitations

anil0225 committed:
Updated documenation with images

anil0225 committed:
Updated table list

anil0225 committed:
Updated suggestions

anil0225 committed:
Renaming to test

anil0225 committed:
Rename back to png

anil0225 committed:
Update domain

mattchenderson committed:
fixing formatting on AAD topic

mattchenderson committed:
refreshing facebook topic

mattchenderson committed:
refreshing facebook topic

mattchenderson committed:
refreshing aad topic

mattchenderson committed:
refreshing twitter topic

mattchenderson committed:
refreshing MSA topic

mattchenderson committed:
refreshing Google topic

v-viinde committed:
Updated

nberdy committed:
Update toc.yml

Adding article on TPM attestation

nberdy committed:
Create concepts-tpm-attestation.md

Stub new article on TPM attestation

nberdy committed:
Update concepts-tpm-attestation.md

nberdy committed:
Create asdf

nberdy committed:
Add files via upload

nberdy committed:
Delete asdf

nberdy committed:
Delete step-one-request-provisioning.png

nberdy committed:
Delete step-three-validation.png

nberdy committed:
Delete step-two-nonce.png

nberdy committed:
Delete tpm-ownership.png

nberdy committed:
Create asdf

nberdy committed:
Add files via upload

nberdy committed:
Delete asdf

Commit:
rework intro, update meta date

Commit:
minor tweak

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-ps-apr20

Commit:
updates

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-cli-apr20

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-cli-apr20

Commit:
minor changes

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-cli-apr20

Commit:
changes to title, intro, and cleanup

dkamstra committed:
Additional information about webhook behavior

Brenduns committed:
fixing create a plan procedures and images

rambk committed:
update ER-VPN coexist doc

Brenduns committed:
fixing create a plan procedures and images

rambk committed:
fixed ER-VPN coexist doc issues

rambk committed:
fixed ER-VPN coexist doc issues

rambk committed:
fixed ER-VPN coexist doc issues

rambk committed:
fixed ER-VPN coexist doc issues

rambk committed:
fixed ER-VPN coexist doc issues

rayne-wiselman committed:
fix couple of issues

rayne-wiselman committed:
fixing Anant's comments

mmacy committed:
[acr] second pass on ACR Build tutorial

parakhj committed:
Adding article for OIDC idp

SnehaGunda committed:
updating landing page

zchia committed:
Update active-directory-saas-cornerstone-ondemand-provisioning-tutorial.md

zchia committed:
Update active-directory-saas-zendesk-provisioning-tutorial.md

SnehaGunda committed:
Adding Gremlin API reference

SnehaGunda committed:
Adding Gremlin API reference

zchia committed:
Update active-directory-saas-replicon-provisioning-tutorial.md

zchia committed:
Update active-directory-saas-bluejeans-provisioning-tutorial.md

anil0225 committed:
Merge pull request #13 from zchia/patch-14

Update active-directory-saas-bluejeans-provisioning-tutorial.md

anil0225 committed:
Merge pull request #11 from zchia/patch-12

Update active-directory-saas-replicon-provisioning-tutorial.md

anil0225 committed:
Merge pull request #10 from zchia/patch-11

Update active-directory-saas-zendesk-provisioning-tutorial.md

anil0225 committed:
Merge pull request #9 from zchia/patch-10

Update active-directory-saas-cornerstone-ondemand-provisioning-tutori…

SnehaGunda committed:
Adding Gremlin API reference

v-viinde committed:
updated

parakhj committed:
Minor edits

v-viinde committed:
Updated

Commit:
freshness

Commit:
automation : add additional example for job output script

mrbullwinkle committed:
Dale's v5 changes

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-cli-apr20

Brenduns committed:
fixing an image

DCtheGeek committed:
Fixing for GH Issue #7568 and minor cleanup

Brenduns committed:
fixing an image

Brenduns committed:
fixing an image

mmacy committed:
[ACR] base image update clarification

mmacy committed:
[ACR] base image requirements

mmacy committed:
[ACR] typo fix

Commit:
minor rewording

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

Brenduns committed:
fixing an image

rayne-wiselman committed:
adding new graphic

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

Commit:
Update analysis-services-vnet-gateway.md

mrbullwinkle committed:
mrb_04_23_2018_pricev5

nberdy committed:
Delete tpm-ownership.png

nberdy committed:
Add files via upload

Commit:
Merge pull request #38663 from mmacy/acr-build-tutorial-update

[ACR] ACR Build tutorial second pass

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

mrbullwinkle committed:
001 change

nberdy committed:
Update concepts-tpm-attestation.md

Updating from reviewer feedback.

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7555

The following example command does not work:
func new --language JavaScript --template HttpTrigger --name MyHttpTrigger
It fails with the exception: Can't find template "HTTPTrigger" in "JavaScript"
The following command does work as is probably what was intended:
func new --language JavaScript --template "HTTP Trigger" --name HTTPTrigger

nberdy committed:
Update concepts-tpm-attestation.md

Commit:
devices02

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#7562

parakhj committed:
Random commit

PRMerger15 committed:
Merge pull request #38707 from MicahMcKittrick-MSFT/patch-21

Fixes MicrosoftDocs/azure-docs#7555

Commit:
Removed classic gw transit constraint.

@anavin

DCtheGeek committed:
Removed version specific Azure PowerShell Links

Commit:
Removed classic GW transit constraint.

@anavin

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix2

mrbullwinkle committed:
Update

GitHubber17 committed:
Merge pull request #38650 from rayne-wiselman/rayne-scenarios-fix

fix couple of issues

mrbullwinkle committed:
update

PRMerger19 committed:
Merge pull request #38625 from dkamstra/master

Additional information about webhook behavior

chrissie926 committed:
IoT Hub module identity module twin articles

PRMerger20 committed:
Merge pull request #38710 from tfitzmac/0423move

fixes MicrosoftDocs/azure-docs#7562

billmath committed:
updating

Commit:
Edits.

GitHubber17 committed:
Merge pull request #38569 from nberdy/patch-2

Update toc.yml

billmath committed:
adding redirector

Commit:
Edits.

v-shils committed:
Merge pull request #38708 from Minewiskan/patch-5

Update analysis-services-vnet-gateway.md

Brenduns committed:
Updated image and Lowercase fix

billmath committed:
removing next steps link

MicrosoftGuyJFlo committed:
[MFA] Cleanup unused media after file moves

Commit:
devices02

mmacy committed:
[ACR] lower-p in TOC

v-shils committed:
Merge pull request #38357 from lnalepa/lesia-v2-app-reg

Updated v2-app-registration doc

Commit:
Merge pull request #38713 from jimdial/patch-2

Removed classic GW transit constraint.

tfitzmac committed:
updated azure cli

chetansriv committed:
Deleting unnecessary files

Brenduns committed:
Updated image and Lowercase fix

tfitzmac committed:
style change

hpsin committed:
Make clear what the TimeSpan format should be. 

For #7551 in the public repo.

v-shils committed:
Merge pull request #38608 from dan-wesley/dawesl-quickwindowsvm-portal-apr20

dawesl-quickwindowsvm-portal-apr20

Commit:
devices02

v-shils committed:
Merge pull request #38722 from mmacy/acr-toc-fix

[ACR] lower-p in TOC

PRMerger14 committed:
Merge pull request #38485 from mattchenderson/authrefresh

App Service Authentication / Authorization IDP refresh

CarlRabeler committed:
[sql db] spark connector architecture

mmacy committed:
[ACR] link to scripts in own section

PRMerger15 committed:
Merge pull request #38728 from hpsin/patch-8

Make clear what the TimeSpan format should be.

v-shils committed:
Merge pull request #38695 from mrbullwinkle/mrb_04_23_2018_pricev5

Dale's v5 changes

PRMerger17 committed:
Merge pull request #38726 from tfitzmac/0423deployops

updated azure cli

ktoliver committed:
Merge pull request #38064 from anil0225/CornerstoneLimiations

Cornestone Limitations - Postion Attribute Mapping

parakhj committed:
Adding a best practice

markgalioto committed:
changing link to the Azure Stack VM size calculator

PRMerger19 committed:
Merge pull request #38712 from jimdial/patch-1

Removed classic gw transit constraint.

PRMerger19 committed:
Merge pull request #38689 from Blackmist/freshness

freshness

v-shils committed:
Merge pull request #38221 from anil0225/bluejeansprovisioning

Bluejeans - Updated Limitations

Commit:
freshness

Commit:
fixing indentation

Commit:
resolve merge

v-shils committed:
Merge pull request #38368 from anil0225/repliconMappingUpdate

Replicon Updated Mappings Image

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-cli-apr20

Commit:
Merge pull request #38736 from parakhj/SSO

Adding a best practice

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#7547

dlepow committed:
Batch: custom image update

JasonWHowell committed:
Update overview.md

Commit:
Merge pull request #38739 from Blackmist/freshness

freshness

Commit:
Update TOC.md

PRMerger10 committed:
Merge pull request #38741 from iainfoulds/patch-2

[VM] Adding username to SSH command

rjagiewich committed:
Merge pull request #38208 from anil0225/zendeskReprovisionWithLimitations

Zendesk Provisioning

v-shils committed:
Merge pull request #38667 from SnehaGunda/master

updating landing page

v-shils committed:
Merge pull request #38628 from Brenduns/quick-fix

fixing create a plan procedures and images

rjagiewich committed:
Merge pull request #38466 from anil0225/xmatterProvisioning

Xmatter Provisioning

v-shils committed:
Merge pull request #38709 from MarkusVi/devices02

devices02

ktoliver committed:
Merge pull request #38665 from parakhj/oidcidp

Adding article for OIDC idp

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quickwindowsvm-cli-apr20

Commit:
revised with Matt's feedback

tynevi committed:
Merge pull request #38727 from chetansriv/customrole

Deleting unnecessary files

v-kents committed:
convert key-vault to yaml

PRMerger18 committed:
Merge pull request #38742 from JasonWHowell/patch-2

Fixing broken bookmark causing build warnings - Update overview.md

MicrosoftGuyJFlo committed:
[Azure AD] Authentication - SSPR Licensing Doc

markgalioto committed:
fixing link

billmath committed:
removing toc entry

billmath committed:
removing links

v-shils committed:
Merge pull request #38730 from CarlRabeler/master

[sql db] spark connector architecture

v-shils committed:
Merge pull request #38744 from Minewiskan/patch-5

Update TOC.md

tdykstra committed:
RequestBin no longer available

Commit:
Merge pull request #38737 from markgalioto/apr232018

changing link to the Azure Stack VM size calculator

itechedit committed:
Merge pull request #38720 from MicrosoftGuyJFlo/MFACleanup

[MFA] Cleanup unused media after file moves

v-kents committed:
correct image links

v-kents committed:
update reference links

mmacy committed:
[ACR] fix busted links

v-kents committed:
use actual image name

curtand committed:
[Azure AD licensing] new script to replace duplication

PRMerger15 committed:
Merge pull request #38743 from dlepow/fix2

Batch: custom image update

PRMerger19 committed:
Merge pull request #38760 from curtand/issues0423

[Azure AD licensing] new script to replace duplication

ktoliver committed:
capitalize Windows Server

ktoliver committed:
capitalize Windows Server

ktoliver committed:
capitalize Windows Server

omidm1 committed:
Removed the note regarding AD on IaaS VM

MicrosoftGuyJFlo committed:
[Azure AD] Authentication SSPR - Group nesting

itechedit committed:
Merge pull request #38718 from billmath/b2b

Removing per engineering.

Commit:
Merge pull request #38749 from MicrosoftGuyJFlo/SSPRLicenseDoc

[Azure AD] Authentication - SSPR Licensing Doc

Commit:
Merge pull request #38752 from tdykstra/patch-39

RequestBin no longer available

PRMerger10 committed:
Merge pull request #38765 from omidm1/master

Removed the note regarding AD on IaaS VM

ktoliver committed:
Merge pull request #38614 from dan-wesley/dawesl-quickwindowsvm-cli-apr20

dawesl quickwindowsvm cli apr20

PRMerger10 committed:
Merge pull request #38768 from MicrosoftGuyJFlo/SSPRNestedGroups

[Azure AD] Authentication SSPR - Group nesting

GitHubber17 committed:
corrected date typo

GitHubber17 committed:
corrected date typo

tynevi committed:
fixing isssue

PRMerger15 committed:
Merge pull request #38694 from georgewallace/autojobstream

automation : add additional example for job output script

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Merge pull request #38701 from DCtheGeek/dmc-policyfixes

Fixing for GH Issue #7568 and minor cleanup

GitHubber17 committed:
Merge pull request #38731 from mmacy/acr-auth-script-promote

[ACR] link to scripts in own section (fixes #7437)

GitHubber17 committed:
Merge pull request #37910 from v-viinde/shmoopmetadataupdate

USER STORY 1237081 SaaS App Tutorial: Shmoop For Schools Metadata URL…

GitHubber17 committed:
Merge pull request #37913 from v-viinde/qprismmetadataupdate

USER STORY 1237076 SaaS App Tutorial: QPrism Metadata URL Update

PRMerger17 committed:
Merge pull request #38733 from chrissie926/release-build-module-twin

IoT Hub module identity module twin articles

v-alje committed:
Merge pull request #38776 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aas0423

tynevi committed:
Merge pull request #38774 from tysonn/master

fixing issue

Commit:
Removed image

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aas0423

rmca14 committed:
Merge pull request #38748 from v-kents/1243420

Convert key-vault to yaml

CarlRabeler committed:
[sql db] removed PITR

fhryo-msft committed:
Update for metrics GA

Update for metrics GA

GitHubber17 committed:
Merge pull request #38781 from Minewiskan/aas0423

Aas0423

PRMerger14 committed:
Merge pull request #38782 from CarlRabeler/createDB

[sql db] removed PITR

GitHubber17 committed:
Merge pull request #38629 from rambk/ERcoexVPN

update ER-VPN coexist doc

PRMerger19 committed:
Merge pull request #38785 from fhryo-msft/master

Update for metrics GA

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
